library/agent/api-discovery/getApiAuthType.ts

```diff
@@ -98,10 +100,20 @@ function getAuthorizationHeaderType(
 function findApiKeys(context: Context): APIAuthType[] {
   const result: APIAuthType[] = [];
 
+  const headerNames = Object.keys(context.headers);
   for (const header of commonApiKeyHeaderNames) {
-    if (context.headers[header]) {
-      result.push({ type: "apiKey", in: "header", name: header });
-    }
+    const matches = headerNames.filter((name) =>
+      name.toLowerCase().includes(header.toLowerCase())
```

Comment:
The headers are already lowercased, so `header.toLowerCase()` shouldn't be necessary, but it doesn't hurt much.

Reply:
done